.github/workflows/cypress-workflow-bundle-snapshot-based.yml

```diff
@@ -33,6 +33,6 @@ jobs:
     uses: ./.github/workflows/release-e2e-workflow-template.yml
     with:
       test-name: Core Dashboards using Bundle Snapshot
-      test-command: env CYPRESS_ML_COMMONS_DASHBOARDS_ENABLED=true CYPRESS_VISBUILDER_ENABLED=true CYPRESS_DATASOURCE_MANAGEMENT_ENABLED=true yarn cypress:run-without-security --browser chromium --spec 'cypress/integration/core-opensearch-dashboards/opensearch-dashboards/**/*.js'
+      test-command: env CYPRESS_ML_COMMONS_DASHBOARDS_ENABLED=true CYPRESS_VISBUILDER_ENABLED=true CYPRESS_DATASOURCE_MANAGEMENT_ENABLED=true yarn cypress:run-without-security --browser chromium --spec 'cypress/integration/core-opensearch-dashboards/opensearch-dashboards/**/*.js' --env NO_COMMAND_LOG=1
```

Comment:
Add the env to last could cause the -env in `cypress:run-with-security` missing, an example https://github.com/opensearch-project/opensearch-dashboards-functional-test/actions/runs/7992090762/job/21824813189

Recommend add this flag to the beginning `env NO_COMMAND_LOG=1 ...`

Reply:
Fixed. Could you help me to take a look?

Reply:
looks good to me, and please revert the changes for testing.